### PR TITLE
added packages/parmap/parmap.1.0-rc11

### PR DESCRIPTION
**Pull request by UnixJunkie**
should compile even with OCaml 4.09.0

### CODE REVIEW

**Comment by mseri** [comment]
You need use `~rc11` Otherwise it will not be considered newer than the final release

**Comment by UnixJunkie** [comment]
@mseri What do you mean? Where should I use '~rc11'?

**Comment by mseri** [comment]
`parmap.1.0~rc11`

**Comment by UnixJunkie** [comment]
I still don't understand. Is this the git tag that should be renamed?

**Comment by mseri** [comment]
No, it’s just the package name: 1.0.0~rc11 < 1.0.0 < 1.0.0-rc11

**Comment by kit-ty-kate** [comment]
The version mistake has already been made and can't be fixed without renaming the previous ones. The current version is fine as is for now

File: packages/parmap/parmap.1.0-rc11/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+dev-repo: "git+https://github.com/rdicosmo/parmap.git"
+bug-reports: "https://github.com/rdicosmo/parmap/issues"
+build: [
+  ["./configure"] 
+  [make "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib" ]
+]
+install: [
+  [make "install" "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib"]
+]
+remove: [
```
Comment (kit-ty-kate) [inline packages/parmap/parmap.1.0-rc11/opam]: The `remove` field is unnecessary since opam 2.0

File: packages/parmap/parmap.1.0-rc11/opam [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  ["./configure"] 
+  [make "uninstall" "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib"]
+]
+depends: [
+  "ocaml" {< "4.08.0"}
```
Comment (kit-ty-kate) [inline packages/parmap/parmap.1.0-rc11/opam]:
```suggestion
  "ocaml" {< "4.08.0"}
```
If the big change of this version is `should compile even with OCaml 4.09.0`, why is the constraint on ocaml still the same?

File: packages/parmap/parmap.1.0-rc11/opam (hunk 1 above)
Comment (UnixJunkie) [reply packages/parmap/parmap.1.0-rc11/opam]: ok, removed

File: packages/parmap/parmap.1.0-rc11/opam (hunk 2 above)
Comment (UnixJunkie) [reply packages/parmap/parmap.1.0-rc11/opam]: I removed the constraint

**Comment by UnixJunkie** [comment]
@mseri I just named the package as the previous ones.
Maybe, the renaming should be made in another PR.
From what I see in the opam-repository, people don't depend on parmap with a specific version constraint on it, so it should not break anything.

**Comment by XVilka** [comment]
@glondu FYI, for your Debian OCaml 4.08+ [transition project](https://wiki.debian.org/Teams/OCamlTaskForce/TODO).

**Comment by UnixJunkie** [comment]
I think the tests are OK.
The broken revdeps are packages of mine that I can fix later (probably by adding an upper bound on their parany library dependency).

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**